### PR TITLE
Created data model for Article and User

### PR DESCRIPTION
**Pull request by vibhorsingh11**
Created data model for Article and User

### CODE REVIEW

File: backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/Article.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Getter
+@Setter
+@ToString
```
Comment (ani03sha) [inline backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/Article.java:9]: We need to have other **Lombok** annotations here - `@EqualsAndHashCode`, `@NoArgsConstructor` and `AllArgsConstructor`,

File: backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@ToString
+public class User {
```
Comment (ani03sha) [inline backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java:13]: We need to have other **Lombok** annotations here - `@EqualsAndHashCode`, `@NoArgsConstructor` and `AllArgsConstructor`,

File: backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    private String userId;
+    private String fullName;
+    private String username;
+    private String email;
+    private String password;
+    private String bio;
+    private String profilePic; // URL of the pic stored in the database
```
Comment (ani03sha) [inline backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java]: Remove the comment.

File: backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/Article.java (hunk 1 above)
Comment (vibhorsingh11) [reply backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/Article.java:9]: fixed

File: backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java (hunk 2 above)
Comment (vibhorsingh11) [reply backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java:13]: Done

File: backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java (hunk 3 above)
Comment (vibhorsingh11) [reply backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java]: removed

**Review by ani03sha — APPROVED** [review]

**Pull request merged**